Assets/MixedRealityToolkit/Definitions/MixedRealityToolkitConfigurationProfile.cs

```diff
@@ -38,16 +40,17 @@ public ExperienceScale TargetExperienceScale
         }
 
         [SerializeField]
-        [Tooltip("Enable the Camera Profile on Startup.")]
-        private bool enableCameraProfile = false;
+        [FormerlySerializedAs("enableCameraProfile")]
+        [Tooltip("Enable the Camera System on Startup.")]
+        private bool enableCameraSystem = false;
 
         /// <summary>
         /// Enable and configure the Camera Profile for the Mixed Reality Toolkit
         /// </summary>
-        public bool IsCameraProfileEnabled
+        public bool IsCameraSystemEnabled
         {
-            get { return CameraProfile != null && enableCameraProfile; }
-            internal set { enableCameraProfile = value; }
+            get { return CameraProfile != null && enableCameraSystem; }
```

Comment:
```suggestion
            get { return CameraProfile != null && cameraSystemType != null && cameraSystemType.Type != null && enableCameraSystem; }
```

Reply:
Is there a way to initialize a system type with a default value? Would be nice if we didn't have to worry about them blowing up.